marketplace/solution/ui_definition.json

```diff
@@ -430,6 +430,73 @@
             "visible": "[equals(steps('scheduler').scheduler, 'slurm')]"
           }
         ]
+      },
+      {
+        "name": "applications",
+        "label": "Applications",
+        "subLabel": {
+          "preValidation": "Applications you want to install",
+          "postValidation": "Done"
+        },
+        "bladeTitle": "Applications",
+        "elements": [        
+          {
+            "name": "enableCodeserver",
+            "type": "Microsoft.Common.CheckBox",
+            "label": "Enable Codeserver",
+            "toolTip": "Enable the Codeserver application",
+            "defaultValue": false,
+            "visible": true
+          },
+          {
+            "name": "enableJupyter",
+            "type": "Microsoft.Common.CheckBox",
+            "label": "Enable Jupyter",
+            "toolTip": "Enable the Jupyter application",
+            "defaultValue": false,
+            "visible": true
+          },
+          {
+            "name": "enableRStudio",
+            "type": "Microsoft.Common.CheckBox",
+            "label": "Enable RStudio",
+            "toolTip": "Enable the RStudio application",
+            "defaultValue": false,
+            "visible": true
+          },
+          {
+            "name": "enableAnsysWorkbench",
+            "type": "Microsoft.Common.CheckBox",
+            "label": "Enable AnsysWorkbench",
+            "toolTip": "Enable the AnsysWorkbench application",
+            "defaultValue": false,
+            "visible": true
+          },
+          {
+            "name": "enableVMD",
+            "type": "Microsoft.Common.CheckBox",
+            "label": "Enable VMD",
+            "toolTip": "Enable the VMD application",
```

Comment:
Same as for Ansys WB as we don't install VMD. Instructions can be found here https://azure.github.io/az-hop/apps/index#visual-molecular-dynamics